src/python/pants/help/help_info_extracter.py

```diff
@@ -178,19 +183,8 @@ def get_option_help_info(self, args, kwargs):
                 metavar = self.compute_metavar(kwargs)
                 display_arg = "{}={}".format(scoped_arg, metavar)
                 if is_list_option(kwargs):
-                    # Show the multi-arg append form.
-                    display_args.append("{arg_str} ({arg_str}) ...".format(arg_str=display_arg))
-                    # Also show the list literal form, both with and without the append operator.
-                    if metavar.startswith('"') and metavar.endswith('"'):
-                        # We quote the entire list literal, so we shouldn't quote the individual members.
-                        metavar = metavar[1:-1]
                     display_args.append(
-                        '{arg}="[{metavar}, {metavar}, ...]"'.format(
-                            arg=scoped_arg, metavar=metavar
-                        )
-                    )
-                    display_args.append(
-                        '{arg}="+[{metavar}, {metavar}, ...]"'.format(
+                        "{arg}=\"['{metavar}', '{metavar}', ...]\"".format(
```

Comment:
Awesome, this is much nicer to quote so that the example actually works when copied into the command line.